Modules/Synchronization/src/Synchronization.Application/Datawallets/Queries/GetModifications/Handler.cs

```diff
@@ -15,15 +19,17 @@ public class Handler : IRequestHandler<GetModificationsQuery, GetModificationsRe
 {
     private readonly IdentityAddress _activeIdentity;
     private readonly IBlobStorage _blobStorage;
+    private readonly ILogger<Handler> _logger;
     private readonly ISynchronizationDbContext _dbContext;
     private readonly IMapper _mapper;
     private readonly BlobOptions _blobOptions;
 
-    public Handler(ISynchronizationDbContext dbContext, IMapper mapper, IUserContext userContext, IBlobStorage blobStorage, IOptions<BlobOptions> blobOptions)
+    public Handler(ISynchronizationDbContext dbContext, IMapper mapper, IUserContext userContext, IBlobStorage blobStorage, IOptions<BlobOptions> blobOptions, ILogger<Handler> logger)
     {
         _dbContext = dbContext;
         _mapper = mapper;
         _blobStorage = blobStorage;
+        _logger = logger;
```

Comment:
Should probably remove the _logger since we no longer use it after daniel silvas' suggestion